cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -23,6 +23,13 @@ rm -rf ./kubernetes/; rm -f kubernetes.tar.gz; rm -f ~/.kube/config
 # if you're working with multiple projects and don't want to repeatedly switch
 # between gcloud config configurations.
 export CLOUDSDK_CORE_PROJECT=<your_project_name>
+
+# To run e2e test locally, make sure "Application Default Credentails" is set in any of the places:
```

Comment:
nit: add a link to the official documentation in case this changes.

Reply:
https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_service_account is the right link for this

Reply:
Thanks @pjh, found this link, is it better? http://cloud/sdk/gcloud/reference/auth/application-default/